client/src/app/[locale]/(admin)/dashboard/config.ts

```diff
@@ -0,0 +1,90 @@
+import * as Yup from 'yup';
+import { TranslationValues } from 'next-intl';
+import { isValidPhoneNumber } from 'libphonenumber-js';
+
+interface FormValues {
+  name: string;
+  site: string;
+  email: string;
+  phone: string;
+  agree: boolean;
+  lastName: string;
+  middleName: string;
+  socialNetworks: string;
+  organizationName: string;
+  positionOrganization: string;
+  organizationTaxNumber: string;
+  certificateOfRegister: string;
+  dateOfRegisterOrganization: string;
+}
```

Comment:
It'd be better to use OOP paradigm here: spit the object into a few other (Organization, Representative) and to use aggregation for those objects.